code/game/gamemodes/gang/gang_items.dm

```diff
@@ -75,6 +76,129 @@
+
+/datum/gang_item/function/backup/get_cost(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+	cost = 50 + gang.gateways * 50
+	var/we = 0
+	var/rival = 0
+	for(var/datum/mind/M in SSticker.mode.get_all_gangsters())
+		if(M.current.stat == DEAD || !M.current.client || M.current.client.is_afk())
+			continue
+		if(is_in_gang(M.current, gang.name))
+			we++
+		else
+			rival++
+			world << "[M.current] is being added as rival"
+	cost += max(-50, round((we/(rival+we))*100 - 33) * 2)
+	return cost
+
+/datum/gang_item/function/backup/spawn_item(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+	var/obj/machinery/gang/backup/bup = new(get_turf(user), gang)
+	bup.G = gang
+
+/datum/gang_item/function/backup/get_cost_display(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+	return "([get_cost(user, gang, gangtool)] Influence)"
+
+/obj/machinery/gang/backup
+	name = "gang reinforcements portal"
+	desc = "When a gang leader leverages their influence, they can pull in some muscle from other operations."
+	anchored = TRUE
+	density = TRUE
+	icon = 'icons/obj/machines/teleporter.dmi'
+	icon_state = "gang_teleporter_on"
+	var/datum/gang/G
+	var/list/queue = list()
+
+/obj/machinery/gang/backup/Initialize(mapload, datum/gang/gang)
+	. = ..()
+	G = gang
+	reinforce()
+	gang.gateways++
+
+
+/obj/machinery/gang/backup/proc/reinforce()
+	if(!src)
+		return
+
+	var/we = 0
+	var/rival = 0
+	var/cooldown = 0
+	for(var/datum/gang/baddies in SSticker.mode.gangs)
+		if(baddies == G)
+			for(var/datum/mind/M in G.gangsters)
+				if(M.current.stat == DEAD)
+					var/mob/O = M.get_ghost(TRUE)
+					if(O)
+						queue += O
+					continue
+				we++
+		else
+			for(var/datum/mind/M in G.gangsters)
+				if(M.current.stat == DEAD)
+					continue
+				rival++
+	spawn_gangster()
+	if(we == 0)
+		we = 1
+	cooldown = 200+((we/(rival+we))*50)**2
+	world << "[cooldown] deciseconds"
+	addtimer(CALLBACK(src, .proc/reinforce), cooldown)
+
+
+/obj/machinery/gang/backup/proc/spawn_gangster()
+	var/mob/living/carbon/human/H = new(src)
+	var/mob/dead/observer/winner
+	var/obj/item/clothing/uniform = G.inner_outfit
+	var/obj/item/clothing/suit/outerwear = new G.outer_outfit(H)
+	outerwear.armor = list(melee = 20, bullet = 35, laser = 10, energy = 10, bomb = 30, bio = 0, rad = 0, fire = 30, acid = 30)
+	outerwear.body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	outerwear.desc += " Tailored for the [G.name] Gang to offer the wearer moderate protection against ballistics and physical trauma."
+	H.equip_to_slot_or_del(new uniform(H), slot_w_uniform)
+	H.equip_to_slot_or_del(outerwear, slot_wear_suit)
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(H), slot_shoes)
+	H.put_in_l_hand(new /obj/item/weapon/gun/ballistic/automatic/surplus/gang(H))
+	H.equip_to_slot_or_del(new /obj/item/ammo_box/magazine/m10mm/rifle(H),slot_l_store)
+	H.equip_to_slot_or_del(new /obj/item/weapon/switchblade(H),slot_r_store)
```

Comment:
I tried, problem is it relies on the gang's unique clothing and I couldn't find a way to get that information in the outfit datum.